addons/stock/models/stock_orderpoint.py

```diff
@@ -138,6 +192,75 @@ def write(self, vals):
                     raise UserError(_("Changing the company of this record is forbidden at this point, you should rather archive it and create a new one."))
         return super().write(vals)
 
+    @api.model
+    def action_open_orderpoints(self):
+        return self._get_orderpoint_action()
+
+    def action_replenish(self):
+        self._procure_orderpoint_confirm(company_id=self.env.company)
+        notification = self._get_replenishment_order_notification()
+        # Forced to call compute quantity because we don't have a link.
+        self._compute_qty()
+        self.filtered(lambda o: o.create_uid.id == SUPERUSER_ID and o.qty_to_order <= 0.0 and self.trigger == 'manual').unlink()
+        return notification
+
+    def action_replenish_auto(self):
+        self.trigger = 'auto'
+        return self.action_replenish()
+
+    def action_snooze(self):
+        self.snooze = True
```

Comment:
snoozed_until? + should be a datime no?

Reply:
oups. Old code not used anymore. It always dispaly the wizard with action action_orderpoint_snooze

Reply:
I forgot to modify it in js thought